sky/global_user_state.py

```diff
@@ -432,7 +426,8 @@ def get_cluster_from_name(
 
 def get_clusters() -> List[Dict[str, Any]]:
     rows = _DB.cursor.execute(
-        'select * from clusters order by launched_at desc')
+        'SELECT * from clusters order by launched_at desc').fetchall()
```

Comment:
Why do we need the `fetchall()` here?

Reply:
This fixes the issue of only one cluster being shown when calling status. I debugged and noticed that without fetchall, only the most recent cluster is returned as a record.

Reply:
cc @concretevitamin . This might be another reason we should not use the global cursor: the problem here is that the iteration for the rows stops after the first step, which is caused by the global cursor being modified by another query in the function `get_cost_for_cluster` called later inside the loop.
https://github.com/skypilot-org/skypilot/blob/8560b34581d5ae3a7f2cb1d2fa062adc37e74c6d/sky/global_user_state.py#L447

Reply:
I think in other functions, we are reading or modifying only one row, so maybe it is not needed?